src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: string): GroupsMap<T> {
+  const result: GroupsMap<T> = {};
+
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
try to avoid using eslint disable, cause it's a bad practice

Reply:
eslint was not OK with for of loop